docs/modules/trino/pages/usage-guide/client-spooling-protocol.adoc

```diff
@@ -0,0 +1,42 @@
+= Client Spooling Protocol
+:description: Enable and configure the Client Spooling Protocol in Trino for efficient handling of large result sets.
+:keywords: client spooling protocol, Trino, large result sets, memory management
+:trino-docs-spooling-url: https://trino.io/docs/476/client/client-protocol.html
+
+The Client Spooling Protocol in Trino is designed to efficiently handle large result sets. When enabled, this protocol allows the Trino server to spool results to external storage systems, reducing memory consumption and improving performance for queries that return large datasets.
+
+For more details, refer to the link:{trino-docs-spooling-url}[Trino documentation on Client Spooling Protocol {external-link-icon}^].
+
+[IMPORTANT]
+====
+The client spooling protocol was introduced in Trino 466 but it only works reliably starting with Trino 476.
+====
+
+== Configuration
+
+The client spooling protocol is disabled by default.
+To enable it, you need to set the `spec.clusterConfig.clientSpoolingProtocol` configuration property as shown below.
+
+[source,yaml]
+----
+spec:
+  clusterConfig:
+    clientProtocol:
+      spooling:
+        location: "s3://spooling-bucket/trino/" # <1>
+        filesystem:
+          s3: # <2>
+            reference: "minio"
```

Comment:
non-blocking: This now differs from the FTE CRD, which has `s3.connection.reference`, here it's `s3.reference`.
*If* we would do the same we have space for other future attributes like s3 error retries, upload size and whatnot.
I let you decide ;)

Reply:
35e7f2cb3dba632ca3b007a4bc1f82089794a06b

Reply:
it bothered me too but it's still inconsistent across ops (hive for example). It is nicer this way though. https://github.com/stackabletech/trino-operator/pull/793/commits/67945ba67001a3fa5139f41f766ac49461176c13